colibris/skeleton/__packagename__/views.py

```diff
@@ -18,14 +21,16 @@
 # @docs(tags=['Users'],
 #       summary='Reveal details about the current user')
 # @response_schema(schemas.UserSchema())
+# @require_any_permission()
```

Comment:
From my point of view I would prefer (again) something to what DRF has: https://www.django-rest-framework.org/api-guide/permissions/#setting-the-permission-policy .
Looks better for me to have the option to set a class as permission handler.

Reply:
@lucifurtun with the latest commits this will be possible:

 * we're now looking for an attribute called `required_permissions` which can be defined inside the class explicitly instead of decorating it
 * we're now passing the handler to the authorization backend so it will be possible to implement a row-level authorization backend (using permission classes, if that's the recommendation)

Reply:
Cool! Sounds good. As for naming, I would prefer when we expect a class to have the attribute name suffixed by `class(es)`, but I admin that's not critical :).

Reply:
I also favor `_class` when something is a class and that's what's going to happen in the PR that will implement this kind of permissions. These simple permissions will only be used as follows:

    class MyView(views.View):
        required_permissions = constants.ADMIN

or

    class MyView(views.View):
        required_permissions = {constants.ALLOW_READ, constants.ALLOW_LIST}

Another commit will come shortly to allow this kind of simplified syntax.